services/function-load-gen/go.mod

```diff
@@ -0,0 +1,18 @@
+module skywalking.apache.org/function-load-gen
+
+go 1.17
```

Comment:
Can you add a job to set up Go with a fixed version, so that it won't break in the future if GitHub Actions automatically install a newer version?

https://github.com/apache/skywalking-showcase/blob/f1a7ad5056ef2268c697c0f37aa9a7daab35ae12/.github/workflows/publish-images.yaml#L39-L40

Reply:
Also we can add license header in `go.mod` ?

Reply:
> Can you add a job to set up Go with a fixed version, so that it won't break in the future if GitHub Actions automatically install a newer version?
> 
> https://github.com/apache/skywalking-showcase/blob/f1a7ad5056ef2268c697c0f37aa9a7daab35ae12/.github/workflows/publish-images.yaml#L39-L40

https://github.com/arugal/skywalking-showcase/blob/faas/services/function-load-gen/Dockerfile#L19
I think the go fixed version is set up in the Dockerfile, right?

Reply:
> Also we can add license header in `go.mod` ?

done